src/System.Private.TypeLoader/src/Internal/Reflection/Execution/AssemblyBinderImplementation.Ecma.cs

```diff
@@ -0,0 +1,194 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using global::System;
+using global::System.IO;
+using global::System.Reflection;
+using global::System.Diagnostics;
+using global::System.Collections.Generic;
+
+using global::System.Reflection.Runtime.General;
+
+using global::Internal.Reflection.Core;
+using global::Internal.Runtime.TypeLoader;
+
+using System.Reflection.PortableExecutable;
+using System.Reflection.Metadata;
+using System.Collections.Immutable;
+
+using System.Reflection.Runtime.Assemblies;
+
+namespace System.Reflection.Runtime.General
+{
+    //
+    // Collect various metadata reading tasks for better chunking...
+    //
+    internal static class EcmaMetadataReaderExtensions
```

Comment:
EcmaMetadataReader extensions moved. PEFile made an internal implementation detail of the assembly binder, and therefore not moved to another file.